Automattic/jetpack
# Only load Mapkit library block once per page

Fixes https://github.com/Automattic/wp-calypso/issues/74040

## Proposed changes:
This PR adds some logic that makes the Mapkit library only load once per page. This PR does this on both the frontend & editor.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

1. Apply this PR
2. Create a page/post
3. Make sure you're using the Mapkit block by pasting this in your console: `document.cookie = "map_provider=mapkit; path=/;";`
4. Open the Network panel
5. Add two Map blocks, add some points, ...
6. You should see only one request for mapkit.js

![CleanShot 2023-03-06 at 11 31 15@2x](https://user-images.githubusercontent.com/528287/223085341-051a4a0c-0dce-409b-926f-a6f3bfdc4a97.png)

Publish your post or page, and view it on the frontend. You should only see 1 request to `mapkit.js`.
